tests/integration_tests/test_mlflow.py

```diff
@@ -132,3 +132,32 @@ def test_tag_truncation(tmpdir: py.path.local) -> None:
 
     my_user_attr = first_run_dict["data"]["tags"]["my_user_attr"]
     assert len(my_user_attr) <= 5000
+
+
+def test_tag_study_user_attrs(tmpdir: py.path.local) -> None:
+    tracking_file_name = "file:{}".format(tmpdir)
+    study_name = "my_study"
+    n_trials = 3
+
+    mlflc = MLflowCallback(tracking_uri=tracking_file_name, tag_study_user_attrs=True)
```

Comment:
How about testing that it's not there when the parameter is `False`?

Reply:
Addressed in https://github.com/optuna/optuna/pull/1935/commits/a80e9a5d5b3fa298e11126a6e17aa7248d870b03